tests/python/test_resolve.py

```diff
@@ -70,7 +70,13 @@ def test_variants(platform, system, variant, current_arch):
     not (sys.platform == 'linux' and machine().lower() == 'x86_64'),
     reason='No variants for this platform and architecture combination',
 )
-@pytest.mark.parametrize('fs', [[None, [hatch.utils.fs]]], indirect=True)
+@(
+    (lambda f: f)
+    # I have no idea why but the module must be reloaded on Python 3.10 and
+    # doing so causes flakiness so we limit to just that version
+    if sys.version_info[:2] == (3, 10)
```

Comment:
I think this is backwards, shouldn't it be `!= (3,10)` if you want to only reload on 3.10?

Reply:
wow late-night commits are not the best, thank you, fixed!

Reply:
The module reloading is so flaky